staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -867,6 +866,18 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 			SelfLinkPathPrefix: selfLinkPrefix,
 			SelfLinkPathSuffix: "/status",
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(features.ServerSideApply) {
```

Comment:
Did we forget to create the fieldmanager before?

Reply:
I'm pretty sure we have a test where we do status update on CR that has the managed fields?

Reply:
No, previously it just took the existing manager from the parent resource.
You can see how we overwrite it for scale to nil. But it was there.
This just creates a new one to pass a different strategy